airflow/cli/commands/remote_commands/config_command.py

```diff
@@ -64,3 +64,349 @@ def get_value(args):
+    ),
+    ConfigChange(
+        section="core",
+        option="non_pooled_task_slot_count",
+        renamed_to=("core", "default_pool_task_slot_count"),
+    ),
+    ConfigChange(
+        section="core",
+        option="dag_concurrency",
+        renamed_to=("core", "max_active_tasks_per_dag"),
+    ),
+    ConfigChange(
+        section="core",
+        option="sql_alchemy_conn",
+        renamed_to=("database", "sql_alchemy_conn"),
+    ),
+    ConfigChange(
+        section="core",
+        option="sql_engine_encoding",
+        renamed_to=("database", "sql_engine_encoding"),
+    ),
+    ConfigChange(
+        section="core",
+        option="sql_engine_collation_for_ids",
+        renamed_to=("database", "sql_engine_collation_for_ids"),
+    ),
+    ConfigChange(
+        section="core",
+        option="sql_alchemy_pool_enabled",
+        renamed_to=("database", "sql_alchemy_pool_enabled"),
+    ),
+    ConfigChange(
+        section="core",
+        option="sql_alchemy_pool_size",
+        renamed_to=("database", "sql_alchemy_pool_size"),
+    ),
+    ConfigChange(
+        section="core",
+        option="sql_alchemy_max_overflow",
+        renamed_to=("database", "sql_alchemy_max_overflow"),
+    ),
+    ConfigChange(
+        section="core",
+        option="sql_alchemy_pool_recycle",
+        renamed_to=("database", "sql_alchemy_pool_recycle"),
+    ),
+    ConfigChange(
+        section="core",
+        option="sql_alchemy_pool_pre_ping",
+        renamed_to=("database", "sql_alchemy_pool_pre_ping"),
+    ),
+    ConfigChange(
+        section="core",
+        option="sql_alchemy_schema",
+        renamed_to=("database", "sql_alchemy_schema"),
+    ),
+    ConfigChange(
+        section="core",
+        option="sql_alchemy_connect_args",
+        renamed_to=("database", "sql_alchemy_connect_args"),
+    ),
+    ConfigChange(
+        section="core",
+        option="load_default_connections",
+        renamed_to=("database", "load_default_connections"),
+    ),
+    ConfigChange(
+        section="core",
+        option="max_db_retries",
+        renamed_to=("database", "max_db_retries"),
+    ),
+    ConfigChange(
+        section="api",
+        option="access_control_allow_origin",
+        renamed_to=("api", "access_control_allow_origins"),
+    ),
+    ConfigChange(
+        section="api",
+        option="auth_backend",
+        renamed_to=("api", "auth_backends"),
+    ),
+    ConfigChange(
+        section="logging",
+        option="enable_task_context_logger",
+        suggestion="Remove TaskContextLogger: Replaced by the Log table for better handling of task log "
+        "messages outside the execution context.",
+    ),
+    ConfigChange(
+        section="metrics",
+        option="metrics_use_pattern_match",
+    ),
+    ConfigChange(
+        section="metrics",
+        option="timer_unit_consistency",
+        suggestion="In Airflow 3.0, the ``timer_unit_consistency`` setting in the ``metrics`` section is "
```

Comment:
If our cli does not support rst, we probably don't need to use "``"

Reply:
Changed it